src/index.ts

```diff
@@ -65,46 +65,67 @@ const indentTheme = EditorView.baseTheme({
   },
 });
 
-function makeBackgroundCSS(entry: IndentEntry, width: number) {
+function makeBackgroundCSS(entry: IndentEntry, width: number, showFirstIndent: boolean) {
   const { level, active } = entry;
 
-  let css = '';
+  const css: string[] = [];
 
-  for (let i = 0; i < level; i++) {
+  for (let i = showFirstIndent ? 0 : 1; i < level; i++) {
     const part =
       active && active - 1 === i
         ? '--indent-marker-active-bg-part'
         : '--indent-marker-bg-part';
 
-    if (i !== 0 && i !== level) {
-      css += ',';
-    }
-
-    css += `var(${part}) ${i * width}.5ch`;
+    css.push(`var(${part}) ${i * width}.5ch`);
   }
 
-  return css;
+  return css.join(',');
 }
 
+type IndentationMarkerConfiguration = {
+  highlightActiveBlock?: boolean
+  showFirstIndent?: boolean
+};
```

Comment:
oh btw: can we add JS Doc comments here for each option so that we can see their descriptions in the editor?